docs/source/release/v6.3.0/direct_geometry.rst

```diff
@@ -5,23 +5,43 @@ Direct Geometry Changes
 .. contents:: Table of Contents
    :local:
 
-Direct Geometry
----------------
+Algorithms
+----------
+New Features
+############
+- :ref:`LoadDNSEvent <algm-LoadDNSEvent>` loads data from DNS PSD detector into an ``EventWorkspace``.
 
-New Algorithms
-##############
+Improvements
+############
+- The default value of monitor peak width multiplier (``MonitorPeakWidthInSigmas``) has been changed from 3 to 7 in :ref:`DirectILLCollectData <algm-DirectILLCollectData>` .
 
-- :ref:`LoadDNSEvent <algm-LoadDNSEvent>` loads data from DNS PSD detector into EventWorkspace
+Bugfixes
+########
+- Updated instrument created by :ref:`LoadNXSPE <algm-LoadNXSPE>` is viewable in instrument 3D view.
 
+CrystalField
+------------
 Improvements
 ############
 - The :ref:`Crystal Field Python interface <Crystal Field Python Interface>` has two new fitting functions, ``two_step_fit`` and ``two_step_fit_sc``, alternating optimization over field parameters and peak parameters. One function is based on the Mantid fitting for both parts, the other uses ``scipy.optimize.minimize`` for the field parameters.
-- The default value of monitor peak width multiplier (``MonitorPeakWidthInSigmas``) has been changed from 3 to 7 in :ref:`DirectILLCollectData <algm-DirectILLCollectData>` .
 - The :ref:`PyChop <PyChop>` GUI now has the ability to handle multiple independently phased choppers.
```

Comment:
The PyChop improvement could be added to 'Interfaces' instead of 'CrystalField'.